sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -455,6 +456,22 @@ object DateTimeUtils {
     (MICROSECONDS.toSeconds(localTimestamp(microsec, timeZone)) % 60).toInt
   }
 
+  /**
+   * Returns seconds, including fractional parts, multiplied by 1000. The timestamp
```

Comment:
@MaxGekk, if https://github.com/apache/spark/pull/25408#discussion_r312748606 matters, we could 1,000 or 1,000,000, I believe.

Reply:
+1 for @HyukjinKwon . My reasons are here, @MaxGekk .
- https://github.com/apache/spark/pull/25408#discussion_r312775959

Reply:
I asked why it is better because @dongjoon-hyun suggested to changed without explanation. Now it is clear for me, and I would keep this as is without gaps.

Reply:
Yep. I'll put more details next time, @MaxGekk ~